pages/api/entity_types.js

```diff
@@ -51,6 +53,9 @@ async function handler(req, res) {
 
   async function get(req, res) { 
     try{
+
+      await assertUserCan([readContentTypes, writeContentTypes], req);
```

Comment:
My intention with the array parameter is for it to be an `OR` assertion (read_contents OR write_contents), as opposed to `await assertUserCan(readContentTypes, req) &&
      await assertUserCan(writeContentTypes, req)`, which is an `AND` assertion.